tests/v2/validation/rbac/clusterandprojectroles/project_role_test.go

```diff
@@ -0,0 +1,119 @@
+	clusterID, err := clusters.GetClusterIDByName(pr.client, clusterName)
+	require.NoError(pr.T(), err, "Error getting cluster ID")
+	pr.cluster, err = pr.client.Management.Cluster.ByID(clusterID)
+	require.NoError(pr.T(), err)
+}
+
+func (pr *ProjectRolesTestSuite) testSetupUserAndProject() (*rancher.Client, *management.Project) {
+	pr.T().Log("Set up User with cluster Role for additional rbac test cases " + rbac.ClusterOwner)
+	newUser, standardUserClient, err := rbac.SetupUser(pr.client, rbac.StandardUser.String())
+	require.NoError(pr.T(), err)
+
+	createProjectAsAdmin, err := pr.client.Management.Project.Create(projects.NewProjectConfig(pr.cluster.ID))
+	require.NoError(pr.T(), err)
+
+	log.Info("Adding a standard user as project Owner in the admin project")
+	errUserRole := users.AddProjectMember(pr.client, createProjectAsAdmin, newUser, rbac.ProjectOwner.String(), nil)
+	require.NoError(pr.T(), errUserRole)
+	standardUserClient, err = standardUserClient.ReLogin()
+	require.NoError(pr.T(), err)
+	return standardUserClient, createProjectAsAdmin
+}
+
+func (pr *ProjectRolesTestSuite) TestProjectOwnerAddsAndRemovesOtherProjectOwners() {
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	standardUserClient, adminProject := pr.testSetupUserAndProject()
+	additionalUser, additionalUserClient, err := rbac.SetupUser(pr.client, rbac.StandardUser.String())
+	require.NoError(pr.T(), err)
+
+	errUserRole := users.AddProjectMember(standardUserClient, adminProject, additionalUser, rbac.ProjectOwner.String(), nil)
+	require.NoError(pr.T(), errUserRole)
+	additionalUserClient, err = additionalUserClient.ReLogin()
+	require.NoError(pr.T(), err)
+
+	userGetProject, err := projects.ListProjectNames(additionalUserClient, pr.cluster.ID)
+	require.NoError(pr.T(), err)
+	assert.Equal(pr.T(), 1, len(userGetProject))
+	assert.Equal(pr.T(), adminProject.Name, userGetProject[0])
+
+	errRemoveMember := users.RemoveProjectMember(standardUserClient, additionalUser)
+	require.NoError(pr.T(), errRemoveMember)
+
+	userProjectEmptyAfterRemoval, err := projects.ListProjectNames(additionalUserClient, pr.cluster.ID)
+	require.Empty(pr.T(), userProjectEmptyAfterRemoval)
+}
+
+func (pr *ProjectRolesTestSuite) TestManageProjectUserRoleCannotAddProjectOwner() {
+	subSession := pr.session.NewSession()
+	defer subSession.Cleanup()
+
+	standardUserClient, adminProject := pr.testSetupUserAndProject()
+	additionalUser, additionalUserClient, err := rbac.SetupUser(pr.client, rbac.StandardUser.String())
+	require.NoError(pr.T(), err)
+
+	errUserRole := users.AddProjectMember(standardUserClient, adminProject, additionalUser, rbac.CustomManageProjectMember.String(), nil)
+	require.NoError(pr.T(), errUserRole)
+	additionalUserClient, err = additionalUserClient.ReLogin()
+	require.NoError(pr.T(), err)
+
+	addNewUserAsPO, addNewUserAsPOClient, err := rbac.SetupUser(pr.client, rbac.StandardUser.String())
```

Comment:
what does 'asP0` mean?

Reply:
PO is ProjectOwner

Reply:
this is very confusing to me, as it looks like 0 😓 I would suggest naming with the full name in the var